plugins/open-feedback/src/components/OpenFeedbackModal/OpenFeedbackModal.tsx

```diff
@@ -20,22 +20,26 @@ import {
   FormControlLabel,
   Checkbox,
   DialogActions,
+  Fab,
 } from '@material-ui/core';
+import { cleanUrl } from '../../utils/cleanUrl';
 
-export type SidebarOpenfeedbackProps = {
+export type ButtonOpenfeedbackProps = {
```

Comment:
Looking at this now, there's no need to export it, and I am not sure we should make a `types.ts` file for the take of one type...

Lets just do `type ButtonOpenfeedbackProps = {...` 

👍

Reply:
thank you for pointing that out! Yes you're right, just changed that: https://github.com/Parsifal-M/backstage-open-feedback/pull/49/commits/59a7d447b9623b238d33f7004bb901a372aef98b